tests/unit/AccountService.js

```diff
@@ -0,0 +1,23 @@
+describe('AccountService tests', function(){
+  var accountService = new AccountService();
```

Comment:
I would recommend creating a new AccountService for each test( put in in beforeEach, this.accountService = new AccountService) to avoid the risk of shared state between the tests.  You need to strive to make the tests independent from each other